docs/README.dev.md

```diff
@@ -1,6 +1,6 @@
 # `s2s` developer documentation
 
-If you're looking for user documentation, go [here](README.md).
+If you're looking for user documentation, go [here](../README.md).
```

Comment:
This link seems to be broken in the sphinx generated documentation ([quick link](https://ai4s2s--10.org.readthedocs.build/en/10/README.dev.html)).